src/main/app/controller/PostController.ts

```diff
@@ -83,7 +85,12 @@ export class PostController<T extends AnyObject> {
   }
 
   protected getEventName(req: AppRequest<T>): string {
-    if (req.session.userCase.applicationType === ApplicationType.SOLE_APPLICATION && req.session.isApplicant2) {
+    if (
+      req.session.userCase.state === State.ConditionalOrderDrafted &&
+      !(req.session.userCase.applicationType === ApplicationType.SOLE_APPLICATION && req.session.isApplicant2)
```

Comment:
how come? won't this will apply to both joint applications and app2?

Reply:
not app2 in sole journey